src/pages/single-product.html

```diff
@@ -1,17 +1,387 @@
+                    <li>
+                        <span>رنگ</span>
+                        <span class="text-black mr-4">صورتی، سبز، بنفش، آبی</span>
+                    </li>
+                    <li>
+                        <span>سایز</span>
+                        <span class="text-black mr-4">۲۰×۲۵، ۱۳×۱۸، ۱۰×۱۵</span>
+                    </li>
+                    <li>
+                        <span>جنس قاب</span>
+                        <span class="text-black mr-4">MDF</span>
+                    </li>
+                    <li>
+                        <span>جنس کاغذ</span>
+                        <span class="text-black mr-4">Glass</span>
+                    </li>
+                </ul>
+
+                <hr class="bg-[#DEE2EEB2] mt-[48px]">
+
+                <div class="flex flex-row gap-16 justify-start mt-[30px]">
+                    <div class="flex flex-col gap-4 items-center">
+                        <img src="./../assets/images/icons/support.png" alt="Support" class="h-[40px]">
+                        <span class="text-[#626E94] text-xs">
+                            پشتیبانی ۲۴ ساعته
+                        </span>
+                    </div>
+                    <div class="flex flex-col gap-4 items-center">
+                        <img src="./../assets/images/icons/badge.png" alt="Support" class="h-[40px]">
+                        <span class="text-[#626E94] text-xs">
+                            قیمت مناسب
+                        </span>
+                    </div>
+                    <div class="flex flex-col gap-4 items-center">
+                        <img src="./../assets/images/icons/shopping-bag.png" alt="Support" class="h-[40px]">
+                        <span class="text-[#626E94] text-xs">
+                            ارسال به تمام کشور
+                        </span>
+                    </div>
+                    <div class="flex flex-col gap-4 items-center">
+                        <img src="./../assets/images/icons/safe-money.png" alt="Support" class="h-[40px]">
+                        <span class="text-[#626E94] text-xs">
+                            بازگشت وجه
+                        </span>
+                    </div>
+                </div>
+
+
+            </div>
+            <!-- ./short info -->
+
+            <!-- purchase box -->
+            <div class="flex flex-col justify-between basis-3/12 bg-[#F5F8FC] p-8">
+
+                <!-- top -->
+                <div class="flex flex-col">
+                    <h3 class="text-[20px] text-black font-bold">انتخاب رنگ</h3>
+                    <div class="flex flex-row gap-3 mt-2 text-[#9CB0B9]">
+                        <button class="flex flex-row gap-2 justify-center items-center bg-white border py-3 px-4">
+                            <span class="rounded-full bg-blue-600 w-[15px] h-[15px] inline-block"></span> <span>آبی</span>
+                        </button>
```

Comment:
Use `<i>` for elements like icons, bullets and etc.

Always beautify your code.

```suggestion
                        <button class="flex flex-row gap-2 justify-center items-center bg-white border py-3 px-4">
                            <i class="rounded-full bg-blue-600 w-[15px] h-[15px] inline-block"></i>
                            <span>آبی</span>
                        </button>
```